staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -100,11 +100,28 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr admission.Attr
 	}
 
 	wg := sync.WaitGroup{}
-	errCh := make(chan error, len(relevantHooks))
+	errCh := make(chan error, 2*len(relevantHooks)) // double the length to handle extra errors for panics in the gofunc
 	wg.Add(len(relevantHooks))
 	for i := range relevantHooks {
 		go func(invocation *generic.WebhookInvocation, idx int) {
+			ignoreClientCallFailures := false
+			// The ordering of these two defers is critical.  The wg.Done will release the parent go func to close the errCh
+			// that is used by the second defer to report errors.  The recovery and error reporting must be done first.
 			defer wg.Done()
+			defer utilruntime.HandleCrash(
```

Comment:
HandleCrash propagates the panic... how is this keeping this goroutine from panicking?

Reply:
> HandleCrash propagates the panic... how is this keeping this goroutine from panicking?

Crash handling is specified (as I recall) with a `ReallyCrash` that I remember wiring to an env var. That's an openshift-only mechanism to avoid panic crash loops isn't it.....

Reply:
Prevented any crash above.